src/component/cancelRes/cancelReservation.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './cancelReservation.css';
+import { Link } from 'react-router-dom';
+
+const button = {
+  backgroundColor: "#3C5E83",
+  border: "none",
+  color: "white",
+  padding: "6px 10px",
+  textAlign: "center",
+  display: "inline-block",
+  marginLeft: "550px",
+  borderRadius: "5px",
+  fontSize: "20px"
+};
```

Comment:
We should move this to the css file we created for this page

Reply:
I second that